README.md

```diff
@@ -83,17 +83,17 @@ impl StatusMessage {
     * `promise_then` -- attaches the callback back to the current contract once the function is executed;
     * `promise_and` -- combinator, allows waiting on several promises simultaneously, before executing the callback;
     * `promise_return` -- treats the result of execution of the promise as the result of the current function.
-    
+
     Follow [examples/cross-contract-high-level](https://github.com/near/near-sdk-rs/tree/master/examples/cross-contract-high-level)
     to see various usages of cross contract calls, including **system-level actions** done from inside the contract like balance transfer (examples of other system-level actions are: account creation, access key creation/deletion, contract deployment, etc).
 
 * **Initialization methods.** We can define an initialization method that can be used to initialize the state of the
-contract.
+contract. NOTE: `#[init_once]` verifies that the contract has not been initialized (the contract state doesn't exist).
 
     ```rust
     #[near_bindgen]
     impl StatusMessage {
-      #[init]
+      #[init_once]
```

Comment:
Should the documentation and an example remain for `#[init]`?

Reply:
The default option for people to use should be `init_once`. We'll keep `init` as is for backward compatibility reasons, but most `new` functions expect `init_once` behavior

Reply:
Can we mark init as obsolete, so that people could migrate?

Reply:
We need to communicate that. If I read an old example and it has `#[init]` and only can read about `#[init_once]` I would get confused. Let's improve the docs and the changelog.